### PR TITLE
regenerate_bindings

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: src/Dialects/14/SPIRV.jl [hunk 1]
```diff
@@ -4502,14 +4502,14 @@ spv.GlobalVariable @var2 bind(1, 2) : !spv.ptr<f32, Uniform>
 spv.GlobalVariable @var3 built_in(\"GlobalInvocationId\") : !spv.ptr<vector<3xi32>, Input>
 ```
 """
-function GlobalVariable(; type, sym_name, initializer=nothing, binding=nothing, descriptorSet=nothing, builtin=nothing, location=Location())
+function GlobalVariable(; type, sym_name, initializer=nothing, location_=nothing, binding=nothing, descriptorSet=nothing, builtin=nothing, location=Location())
```
Comment (vchuravy) [inline src/Dialects/14/SPIRV.jl:4505]: It's slightly odd that we have two location keywords

Comment (jumerckx) [reply src/Dialects/14/SPIRV.jl:4505]: Agreed. Maybe the "regular" operation location should be type constrained to `IR.Location`, there's no good reason I didn't do this yet.
Or would you prefer handling this by using a different name for the attribute, `location_attribute` or something?

Comment (vchuravy) [reply src/Dialects/14/SPIRV.jl:4505]: No strong opinion I will leave that up to you.

**Comment by vchuravy** [comment]
@mofeing I did have the Manifest.toml for bindings here xD

This PR is run with the newest binding generator but it created not fully functional bindings x-ref https://github.com/JuliaLabs/MLIR.jl/pull/55

**Pull request merged**